docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -379,6 +379,81 @@ I will also verify that the household
+and we will address those separately: children under the age of 18, and 
+those who are below 24 and in GQ for college (defined above). 
+
+Note: "N/A" for the purposes of this simulation means that a parent/
+guardian cannot be identified. For tax purposes, no one will claim 
+this person as a dependent. 
+
+**For simulant under 18:**
+
+- Child is a biological or adopted child to reference person 
+    * Assign reference person 
+- Child is foster child 
+    * Assign reference person 
+- Child is step child 
+    * If there is a spouse / partner of reference person assign them 
+    * Otherwise assign the reference person 
+- Child is grandchild of reference person 
+    * If there is a child (biological, adopted, foster) of reference person, assign them (if multiple, assign at random) 
+    * Otherwise assign reference person 
+- Child is brother/sister to reference person 
+    * If there is a parent of the reference person, assign them 
+    * Otherwise assign the reference person 
+- Child is other relative to reference person 
+    * Assign a relative of the reference person who is between 20 and 45 years older than the child. If there are multiple, select one of the same race/ethnicity. If there are multiple of the same race/ethnicity, assign at random. 
+    * If there is not a relative of the appropriate age available, assign the reference person 
+- Child is non-relative (roommate or other nonrelative) to reference person 
+    * Assign another non-relative of the reference person who is between 20 and 45 years older than the child. If there are multiple, select one of the same race/ethnicity. If there are multiple with the same race/ethnicity, assign at random.  
+    * If there is not a non-relative of the appropriate age available, assign to a non-relative of any age (select at random if multiple) 
+    * If there are not any other non-relatives in the house, make "N/A"
+- Child is the reference person 
+    * Assign a parent, if available 
+    * Otherwise, assign another relative who is between 20 and 45 years older than the child. If there are multiple, select one of the same race/ethnicity. If there are multiple of the same race/ethnicity, assign at random.
+    * If there are no other relatives in the house, make "N/A"
+
+Once the parent/guardian is assigned, if there is a spouse or unmarried partner 
+for that simulant (reference person and spouse/unmarried partner ONLY), then 
+include both as parents/guardians. Otherwise only include the one as a parent/guardian. 
+
+(note to software engineers: if any of these rules turn out to be surprisingly hard to implement, please be in touch with research --- we have some flexibility in just how we do this!)
+
+**For a simulant who is below 24 and in GQ at college:**
+
+Simulant will be randomly assigned to a parent/guardian based on the below rules: 
+
+- 78.5% will be assigned to a parent/guardian within their state. The remainder will be assigned out of state source1_  
```

Comment:
for now, we have only ever run for Florida. Until we run for multiple states, would it be appropriate to ignore those remaining 21.5%?

Reply:
Good point! I will add something about this. I think we can just ignore for now, yes.